app/components/categories.js

```diff
@@ -1,12 +1,16 @@
 import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
 import { A } from '@ember/array';
 import { action } from '@ember/object';
 import { inject as service } from '@ember/service';
 
 
 export default class CategoriesComponent extends Component {
-  store = service();
-  router = service();
+  @service store;
```

Comment:
I saw that this class was using some legacy / classic / in-between stuff? I wasn't sure, but I updated it to modern syntax

Reply:
Might be, I referred to former side projects that were in between Octane updates... Thanks!!